packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -713,6 +733,38 @@ class RenderEditable extends RenderBox {
     markNeedsLayout();
   }
 
+  ///{@template flutter.rendering.editable.paintCursorOnTop}
+  /// If the cursor should be painted on top of the text or underneath it.
+  ///
+  /// By default, the cursor will be painted on top for iOS platforms and
+  /// underneath for Android platforms.
+  /// {@end template}
+  bool get paintCursorAboveText => _paintCursorOnTop;
+  bool _paintCursorOnTop;
+  set paintCursorAboveText(bool value) {
+    if (_paintCursorOnTop == value)
+      return;
+    _paintCursorOnTop = value;
+    markNeedsLayout();
+  }
+
+  /// {@template flutter.rendering.editable.cursorOffset}
+  /// The offset that is used, in pixels, when painting the cursor on screen.
+  ///
+  /// By default, the cursor will be painted on with an offset of
```

Comment:
Saying "will" makes it sound like this render object is doing it, which it isn't as far as I can tell. If you meant "should be set to" then say that. Tweak the doc templating as needed.